packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -58,7 +58,7 @@ const gcOptionsMatrix: OptionsMatrix<IGCRuntimeOptions> = {
 };
 
 const summaryOptionsMatrix: OptionsMatrix<ISummaryRuntimeOptions> = {
-    disableIsolatedChannels: booleanCases,
+    disableIsolatedChannels: [undefined],
```

Comment:
This looks unrelated and should be removed from this PR, right?

Reply:
yes, sorry, forgot to merge.